src/recipe/Baked.Recipe.Service.Application/Cors/AspNetCore/AspNetCoreCorsExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Baked.Cors;
+using Baked.Cors.AllowOrigin;
+using Baked.Runtime;
+using Microsoft.AspNetCore.Cors.Infrastructure;
+
+namespace Baked;
+
+public static class AspNetCoreCorsExtensions
+{
+    /// <summary>
+    /// Returns 'AspNetCoreCors' feature with a single policy setup with given origins,
+    /// any header and any method
+    /// </summary>
+    /// <param name="configurator"></param>
+    /// CorsConfigurator
+    /// <param name="origins">
+    /// Allowed origin addresses
+    /// </param>
+    /// <returns></returns>
```

Comment:
never had xml comments in any of the features. if you have a note use `<note>` tag, otherwise it would create inconsistency and bring the question of "Are we adding xml comments in this feature?" every time we write a feature :)